frontend/src/components/Pages/InspectionPage/InspectionTable.tsx

```diff
@@ -225,21 +270,26 @@ export const InspectionTable = ({ deck, inspections, openDialog, setSelectedMiss
 
     return (
         <StyledTable>
-            <Table>
-                <Table.Caption>
-                    <Typography variant="h3" style={{ marginBottom: '14px' }}>
-                        {TranslateText('Inspection Missions') + ' ' + TranslateText('for') + ' ' + deck.deckName}
-                    </Typography>
-                </Table.Caption>
-                <Table.Head sticky>
-                    <Table.Row>
-                        {columns.map((col) => (
-                            <Table.Cell key={col}>{TranslateText(col)}</Table.Cell>
-                        ))}
-                    </Table.Row>
-                </Table.Head>
-                <Table.Body>{cellValues}</Table.Body>
-            </Table>
+            <HideColumnsOnSmallScreen>
```

Comment:
If you change this to line 179 won't that work too? and then you onlly need it in one place

Reply:
Headers and SmallScreenInfoText won't get hidden if it's moved